src/components/CharacterAssignment.jsx

```diff
@@ -58,15 +70,13 @@ const CharacterAssignment = ({ actors, currentScriptId }) => {
    * @param  {object} newColumns Object of actor column objects, no assigned characters
    * @returns {object} newColumns: Object of actor column objects, now updated with assigned characters
    */
-  const assignInitialCharacters = (characters, newColumns) => {
+  const assignInitialCharacters = (newColumns, characters) => {
```

Comment:
What about potentially dropping the `new` prefix here too?

Reply:
Also, for the @returns, it shouldn't have the variable name listed since that won't be visible to whoever calls this function